src/shapes/Polyline.ts

```diff
@@ -163,15 +163,20 @@ export class Polyline<
       bboxNoStroke = makeBoundingBoxFromPoints(
         this.points.map((p) => transformPoint(p, matrix, true))
       ),
-      offsetX = bbox.left + bbox.width / 2,
-      offsetY = bbox.top + bbox.height / 2,
-      pathOffsetX = offsetX - offsetY * Math.tan(degreesToRadians(this.skewX)),
-      pathOffsetY =
-        offsetY - pathOffsetX * Math.tan(degreesToRadians(this.skewY)),
       scale = new Point(this.scaleX, this.scaleY);
+    let offsetX = bbox.left + bbox.width / 2,
+      offsetY = bbox.top + bbox.height / 2;
+    if (this.exactBoundingBox) {
+      // Order of those assignments is important.
+      // The second assignment relies on offsetX being already changed
```

Comment:
I think it is better to put the comment above offsetY stating that it relies on the new offsetX or introduce more variables to make this clear